cefbrowser/cefbrowser.py

```diff
@@ -142,29 +152,32 @@ def __init__(self, url="", *largs, **dargs):
         self.register_event_type("on_js_dialog")
         self.register_event_type("on_before_unload_dialog")
 
-        self._texture = Texture.create(size=self.size, colorfmt="rgba", bufferfmt="ubyte")
+        self._texture = Texture.create(
+            size=self.size, colorfmt="rgba", bufferfmt="ubyte")
         self._texture.flip_vertical()
         with self.canvas:
             Color(1, 1, 1)
-            self.__rect = Rectangle(pos=self.pos, size=self.size, texture=self._texture)
+            self.__rect = Rectangle(
+                pos=self.pos, size=self.size, texture=self._texture)
 
         if not CEFBrowser._cefpython_initialized:
             cefpython_initialize(CEFBrowser)
             CEFBrowser._cefpython_initialized = True
         if not self._browser:
             # On x11 input provider we have the window-id (handle)
-            wid = 0
+            window_id = 0
             try:
-                from kivy.core.window import Window as kivy_window
-                wid = kivy_window.window_id
+                from kivy.core.window import Window as KivyWindow
+                window_id = KivyWindow.window_id
             except Exception as e:
-                Logger.debug("Use window handle %s, because: %s" % (wid, e))
-            windowInfo = cefpython.WindowInfo()
-            windowInfo.SetAsOffscreen(wid)
+                Logger.debug("Use window handle %s, because: %s", window_id, e)
+            window_info = cefpython.WindowInfo()
+            window_info.SetAsOffscreen(window_id)
```

Comment:
There have been modifications in cefpython's Print functionality. Printing should now work even when parent window handle is not provided in SetAsOffscreen. However I don't remember if window handle was also required for other reasons, whether it is required for mouse context menu and popup widgets to work correctly.